<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowTitle">
   <string>Form - toplu2.ui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textAlan"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnSec">
       <property name="text">
        <string>Yazdırılacak Soru Bankasını Seçiniz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnYazdir">
       <property name="text">
        <string>Yazdır</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>